app/assets/stylesheets/sidebar_admin.css.scss

```diff
@@ -22,7 +22,7 @@
   padding: 45px;
   border: 1px solid #ccc;
 }
-.sidebar_item {
+.sidebar-item {
```

Comment:
Please explain how modifying `sidebar_style` to `sidebar-style` is relevant to original issue.